### PR TITLE
fix(config): define association groups for multiple Jasco devices

### PR DESCRIPTION
**Pull request by disforw**
Adding assiociation groups for the following Jasco devices: 14287, 26932, 45604, 14289, 14292

### CODE REVIEW

File: packages/config/config/devices/0x0063/26932_26933_zw3008.json [hunk 1]
```diff
@@ -24,12 +24,14 @@
...
+			"label": "Group 2 Basic Set",
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled by pressing On/Off button with local load"
 		},
 		"3": {
-			"label": "Group 3",
-			"maxNodes": 5
+			"label": "Group 3 Basic Set",
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled by pressing On/Off button with local load"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/26932_26933_zw3008.json]: What's the difference between groups 2 and 3 here? Or is this a 2-channel device?

File: packages/config/config/devices/0x0063/45604.json [hunk 2]
```diff
@@ -20,5 +20,24 @@
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/45604.json]: We don't use that anymore.
```suggestion
```

File: packages/config/config/devices/0x0063/45604.json [hunk 3]
```diff
@@ -20,5 +20,24 @@
...
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic - Local Load",
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled with the local load."
+		},
+		"3": {
+			"label": "Basic - Double Tap", // required
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/45604.json]:
```suggestion
			"label": "Basic - Double Tap",
```
Please remove these comments

File: packages/config/config/devices/0x0063/45604.json [hunk 4]
```diff
@@ -20,5 +20,24 @@
...
+			"maxNodes": 5,
+			"isLifeline": true,
+			"description": "Supports Basic Set and is controlled by double pressing on/off button."
+		}
 	}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/45604.json:42]: The double-tap function requires the compat flag [`treatBasicSetAsEvent`](https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=treatbasicsetasevent)

**Review by AlCalzone — CHANGES_REQUESTED** [review]
I'm not overly happy with the descriptions. They could be shortened a bit (see also my other PR review) and made clearer - the Basic Set group is to control other devices, Double Tap can additionally be used to notify the controller.

File: packages/config/config/devices/0x0063/26932_26933_zw3008.json (hunk 1 above)
Comment (disforw) [reply packages/config/config/devices/0x0063/26932_26933_zw3008.json]: Must be dual channel, documentation attached, do you want me to just use one? Or leave both?
951523AF-F5FA-4181-8087-5EDF9D04E724

**Review by disforw — COMMENTED** [review]
Removed “supportsZwavePlus” as per comments.
Added compat where needed.
Removed clutter comments.

File: packages/config/config/devices/0x0063/26932_26933_zw3008.json (hunk 1 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0063/26932_26933_zw3008.json]: Might also be On-Button => Group 2 and Off-Button => Group 3. Let's see if we can find someone to confirm.

File: packages/config/config/devices/0x0063/26932_26933_zw3008.json [hunk 5]
```diff
@@ -24,12 +24,14 @@
 			"isLifeline": true
 		},
 		"2": {
-			"label": "Group 2",
-			"maxNodes": 5
+			"label": "Group 2 Basic Set",
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled by pressing On/Off button with local load"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/26932_26933_zw3008.json]:
```suggestion
			"description": "Controlled by pressing On/Off button with local load"
```

File: packages/config/config/devices/0x0063/26932_26933_zw3008.json (hunk 1 above)
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/26932_26933_zw3008.json]:
```suggestion
			"description": "Controlled by pressing On/Off button with local load"
```

File: packages/config/config/devices/0x0063/45604.json [hunk 6]
```diff
@@ -20,5 +20,26 @@
...
 		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic - Local Load",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/45604.json]:
```suggestion
			"label": "Basic Set - Local Load",
```

File: packages/config/config/devices/0x0063/45604.json [hunk 7]
```diff
@@ -20,5 +20,26 @@
...
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled with the local load."
+		},
+		"3": {
+			"label": "Basic - Double Tap",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/45604.json]:
```suggestion
			"label": "Basic Set - Double Tap",
```

File: packages/config/config/devices/0x0063/45604.json [hunk 8]
```diff
@@ -20,5 +20,26 @@
...
+			"maxNodes": 5,
+			"isLifeline": true,
+			"description": "Supports Basic Set and is controlled by double pressing on/off button."
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/45604.json]:
```suggestion
			"description": "Controlled by double pressing on/off button."
```

File: packages/config/config/devices/0x0063/45604.json [hunk 9]
```diff
@@ -20,5 +20,26 @@
...
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic - Local Load",
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled with the local load."
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/45604.json]:
```suggestion
			"description": "Controlled with the local load."
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
A few more consistency nits. Also the group labels are not consistent with each other - I'll let @blhoward2 decide on the desired format.

**Comment by disforw** [comment]
I changed the group names to match, only not sure how to handle the dual channel basic set one.

**Review by disforw — COMMENTED** [review]
All group labels were renamed, and zwaveplus switches removed.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Please make the associations consistent (or remove the descriptions). In 3 files, I see 3 separate descriptions for the same group:
* Controlled with the local load
* Basic Set and is controlled with the local load.
* Supports Basic Set and is controlled by pressing On/Off button with local load.

**Review by disforw — COMMENTED** [review]
Changes as requested

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks

**Pull request merged**